debezium-testing/debezium-testing-system/pom.xml

```diff
@@ -20,7 +20,7 @@
     <version.strimzi.kafka>${version.kafka}</version.strimzi.kafka>
     <version.junit5>5.7.2</version.junit5>
     <version.assertj>3.11.1</version.assertj>
-    <version.apicurio>2.2.0.Final</version.apicurio>
+    <version.apicurio>2.4.1.Final</version.apicurio>
```

Comment:
@jcechace Could we do the change here? I think this is downstream not upstream version, right?

Reply:
@jpechane - Looking at this closer, it appears that this property does not actually control an apicurio dependency.   Were you wanting me to revert this?

Reply:
Reverted by me.